src/microbe_stage/systems/MicrobeReproductionSystem.cs

```diff
@@ -368,6 +368,13 @@ protected override void PostUpdate(float state)
             CalculateFreeCompoundsAndLimits(gameWorld!.WorldSettings, organelles.HexCount, false,
                 reproductionDelta);
 
+        if (entity.Has<MicrobeControl>())
+        {
+            // Microbe can't passively absorb compounds with mucocyst active
+            if (entity.Get<MicrobeControl>().State == MicrobeState.MucocystShield)
+                remainingFreeCompounds = 0;
+        }
+
```

Comment:
I think it is better to disallow all reproduction system running when in mucocyst mode than specifically here (also need to do the same change for multicellular growth system). Also if this system didn't use MicrobeControl before the attributes need t obe adjusted.

Reply:
For muticellular, one cell might have mucocyst while other don't; is it necessary?

Reply:
The main cell does reproduction only in multicellular, and it will call this base reproduction process method.